app/javascript/elm/src/Main.elm

```diff
@@ -612,8 +618,8 @@ view model =
 
 
 viewSearchAsIMove : Bool -> Bool -> Html Msg
-viewSearchAsIMove wasMapMoved isSearchOn =
-    div []
+viewSearchAsIMove wasMapMoved isSearchAsIMoveOn =
+    div [ Html.Attributes.style "position" "absolute", Html.Attributes.style "z-index" "99999" ]
```

Comment:
this already changed on experimental I guess there will be a conflict here, the new version is 
`div [ class "search-control" ]`